front/src/applications/editor/tools/trackSplit/components/TrackSplitLeftPanel.tsx

```diff
@@ -100,8 +100,10 @@ const TrackSplitLeftPanel: FC<unknown> = () => {
                 : undefined
             }
             onChange={(e) => {
-              if (!Number.isNaN(e.target.valueAsNumber))
-                setState((prev) => ({ ...prev, offset: e.target.valueAsNumber * 1000 }));
+              setState((prev) => ({
+                ...prev,
+                offset: (!Number.isNaN(e.target.valueAsNumber) ? e.target.valueAsNumber : 0) * 1000,
```

Comment:
As discussed with @theocrsb, the value can be NaN if the user enters nothing. For improved readability, you can write:
```suggestion
                offset: (e.target.valueAsNumber ?? 0) * 1000,
```
We use the [nullish coalesing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing)

Reply:
Never mind nullish coalesing is not applyed on NaN
https://github.com/tc39/proposal-nullish-coalescing/issues/28